### DISCUSSION TITLE
Generating a Claim Code

### DISCUSSION THREAD

**Discussion by dill-wood** [doitandbedone/ispyagentdvr-docker#608] [Q&A]
Good Morning All and thank you in advance for your time.

I have a local installation of ispy running on Rocky Linux 8. It is strictly a command line gui, so the claim code seems to be not working as intended. I navigated to the ispy how-to guides, and noted there is a script in the docker container that can be leveraged to generate a claim code to link accounts. However, When attempting to run the command inside the docker container, I am met with the following error code:

root@901dcd8467fe:/# systemctl stop AgentDVR.service
System has not been booted with systemd as init system (PID 1). Can't operate.
Failed to connect to bus: Host is down

Thoughts?

Thank you,

**Comment by ispysoftware** [comment]
the docker container should be available over the network so just connect a computer with a web browser and access it over the local network using http://ipaddress:8090 and click on server icon - remote access
